### ISSUE TITLE
Invoke-Shellcode working with updated msf?

### ISSUE THREAD

**Issue by ChrisTruncer** [mattifestation/PowerSploit#39]
Hey there,

First off, big fan of a lot of your scripts on here.  Really useful stuff.  Myself and two other guys work on Veil, and we ran into an issue with powershell injection, and I think it may apply here too.  I could be doing it wrong, but figured I'd at least ask and see your thoughts.

It looks like a recent update to MSF may have caused some problems with injection shellcode via powershell, either into itself, or another process.  We tested powershell injection with an old version of MSF, and it worked, and then testing with the latest version, crashes for us.  We originally thought that it may be a problem on our end, so to test that, I grabbed your Invoke-Shellcode script, formatted the msf shellcode properly for your script, and then tried injecting the shellcode into powershell, and then also into a notepad process I had running, using Invoke-Shellcode.  Both attempts resulted in a crash of the process.

So, long story short, I COULD be doing it wrong, but never seemed to have an issue before when using Invoke-Shellcode, but it seems to crash now.  I was curious if you were also able to re-create the crash.  If you need more info from me, let me know in here, but I didn't do anything special besides giving it -Shellcode and then the shellcode (formatted as per your example in the script).
